app/views/pages/documentation.html.erb

```diff
@@ -64,7 +65,7 @@
     <%= image_tag("AUK_analyze.png", alt: "Analyze the Collection", class:"body_img")%>
 
     <h3 class="about_h3">Output Files</h3>
-    <p class="about_p">You will receive an email once your collections are ready for analysis. When you navigate to the collection space, by clicking on the collection title, (in the screenshot below we've selected the Halifax Explosion collection) you will find:</p>
+    <p class="about_p">You will receive an email once your collections are ready for analysis. When you navigate to the collection space, by clicking on the collection title, (in the screenshot below we've selected the Nova Scotia Municipal Governments collection) you will find:</p>
```

Comment:
Minor super pedantic change (that's a bit out of scope for PR), move the comma until after the parenthetical statement, so it reads

`When you navigate to the collection space, by clicking on the collection title (in the screenshot below we've selected the Nova Scotia Municipal Governments collection), you will find`